webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path')
+
+module.exports = {
+    entry: {
+        fido2: [
+            'babel-polyfill',
+            path.resolve(__dirname, 'src/fido2.js'),
```

Comment:
I'd very much prefer having the JS in `assets` directory rather than `src` directory. There is a Python workflow and packaging convention, that puts all Python modules inside the `src` directory. And even if we don't follow that convention right now, we may decide to do so in the future. Therefore, using non-conflicting name seems to be a better future-proof solution.

Reply:
Fixed